tools/mo/openvino/tools/mo/front/mxnet/layer_norm_ext.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+from openvino.tools.mo.front.extractor import FrontExtractorOp
+from openvino.tools.mo.front.mxnet.extractors.utils import get_mxnet_layer_attrs
+from openvino.tools.mo.graph.graph import Node
+from openvino.tools.mo.ops.layer_norm import LayerNorm
+
+
+class LayerNormFrontExtractor(FrontExtractorOp):
+    op = 'LayerNorm'
+    enabled = True
+
+    @classmethod
+    def extract(cls, node: Node):
+        attr = get_mxnet_layer_attrs(node.symbol_dict)
```

Comment:
nit: please add a short description about arguments and attributes of this operation in MXNet